lib/parsers/project-assets-json-parser.ts

```diff
@@ -0,0 +1,78 @@
+import * as _ from 'lodash';
+
+export interface PkgTree {
+  name: string;
+  version: string;
+  dependencies: {
+    [dep: string]: PkgTree;
+  };
+  targetFrameworks?: string[];
+}
+
+export interface ProjectAssetsJsonManifest {
+  targets: {
+    [target: string]: {
+      [name: string]: {
+        type: string,
+        dependencies?: {
+          [deps: string]: string;
+        };
+      };
+    },
+  };
+  project: {
+    restore: {
+      projectName: string,
+    },
+    version: string,
+  };
+}
+
+export function getProjectNameForProjectAssetsJson(manifestFile) {
+  return _.get(manifestFile, ['project', 'restore', 'projectName'], {});
+}
+
+export function getProjectVersionForProjectAssetsJson(manifestFile) {
+  return _.get(manifestFile, ['project', 'version'], {});
+}
+
+function buildRootPackageTree(name, version) {
+  const depTree: PkgTree = {
+    dependencies: {},
+    name,
+    version,
+  };
+  return depTree;
+}
+
+function buildPackageTreeForDep(name, version) {
+  const topLevelDepTree: PkgTree = {
+    dependencies: {},
+    name,
+    version,
+  };
+  return topLevelDepTree;
+}
+
+export function getDependencyTreeFromProjectAssetsJson(manifestFile: ProjectAssetsJsonManifest,
+                                                       targetFrameWork): PkgTree {
```

Comment:
This long space looks bit weird :-)

Reply:
The linter enforced as it was over max-line length :(

Reply:
You can change it to be
```suggestion
export function getDependencyTreeFromProjectAssetsJson(
    manifestFile: ProjectAssetsJsonManifest,
    targetFrameWork,
): PkgTree {
```

Reply:
👍 updated